docs/content/envoy-authorization.md

```diff
@@ -0,0 +1,334 @@
+            - name: envoy.http_connection_manager
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+                codec_type: auto
+                stat_prefix: ingress_http
+                route_config:
+                  name: local_route
+                  virtual_hosts:
+                    - name: backend
+                      domains:
+                        - "*"
+                      routes:
+                        - match:
+                            prefix: "/"
+                          route:
+                            cluster: service
+                http_filters:
+                  - name: envoy.ext_authz
+                    config:
+                      with_request_body:
+                        max_request_bytes: 8192
+                        allow_partial_message: true
+                      failure_mode_allow: false
+                      grpc_service:
+                        google_grpc:
+                          target_uri: 127.0.0.1:9191
+                          stat_prefix: ext_authz
+                        timeout: 0.5s
+                  - name: envoy.router
+                    typed_config: {}
+  clusters:
+    - name: service
+      connect_timeout: 0.25s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: service
+        endpoints:
+          - lb_endpoints:
+              - endpoint:
+                  address:
+                    socket_address:
+                      address: 127.0.0.1
+                      port_value: 8080
+admin:
+  access_log_path: "/dev/null"
+  address:
+    socket_address:
+      address: 0.0.0.0
+      port_value: 8001
+```
+
+Create the ConfigMap:
+
+```bash
+kubectl create configmap proxy-config --from-file envoy.yaml
+```
+
+The OPA configuration below sets the listening address of Envoy External Authorization gRPC server and specifies the name of the policy decision to query. More information on the configuration options can be found [here](https://github.com/open-policy-agent/opa-istio-plugin#configuration).
+
+Save the configuration as **opa.yaml**:
+
+```yaml
+plugins:
+  envoy_ext_authz_grpc:
+    addr:    :9191
+    query:   data.envoy.authz.allow
+    dry-run: false
+decision_logs:
+    console: true
+```
+
+Create the ConfigMap:
+
+```bash
+kubectl create configmap opa-config --from-file opa.yaml
+```
+
+### 3. Define a OPA policy
+
+The following OPA policy restricts access to the `/people` endpoint exposed by our sample app:
+
+* Alice is granted a **guest** role and can perform a `GET` request to `/people`.
+* Bob is granted an **admin** role and can perform a `GET` and `POST` request to `/people`.
+
+The policy also restricts an `admin` user, in this case `bob` from creating an employee with the same `firstname` as himself.
+
+Store the policy in Kubernetes as a Secret.
+
+**policy.rego**
+
+```ruby
```

Comment:
Let's make use of the live blocks here. The Kafka tutorial has a good example.

Reply:
Updated to use live docs.